src/RA_AchievementOverlay.cpp

```diff
@@ -15,30 +15,24 @@
 #include <memory>
 #include <ctime>
 
-namespace {
-
-const float PAGE_TRANSITION_IN = (-0.2f);
-const float PAGE_TRANSITION_OUT = (0.2f);
-const int NUM_MESSAGES_TO_DRAW = 4;
-const char* FONT_TO_USE = "Tahoma";
-
-const char* PAGE_TITLES[] ={
-    " Achievements ",
-    " Friends ",
-    " Messages ",
-    " News ",
-    " Leaderboards ",
-    " Achievement Info ",
-    " Achievement Compare ",
-    " Friend Info ",
-    " Friend Add ",
-    " Leaderboard Examine ",
-    " Message Viewer "
+namespace ra {
 
+inline constexpr std::array<LPCTSTR, 11> PAGE_TITLES
+{
+    _T(" Achievements "),
+    _T(" Friends "),
+    _T(" Messages "),
+    _T(" News "),
+    _T(" Leaderboards "),
+    _T(" Achievement Info "),
+    _T(" Achievement Compare "),
+    _T(" Friend Info "),
+    _T(" Friend Add "),
+    _T(" Leaderboard Examine "),
+    _T(" Message Viewer ")
 };
```

Comment:
I moved all the other constants somewhere else, `NUM_MESSAGES_TO_DRAW `is gone because it wasn't used.